src/cdk/app.ts

```diff
@@ -1,61 +1,82 @@
-import apigateway = require('@aws-cdk/aws-apigateway');
-import * as iam from '@aws-cdk/aws-iam';
+import { Rule, Schedule } from '@aws-cdk/aws-events';
+import { LambdaFunction } from '@aws-cdk/aws-events-targets';
+import { Policy, PolicyStatement } from '@aws-cdk/aws-iam';
 import lambda = require('@aws-cdk/aws-lambda');
-import s3 = require('@aws-cdk/aws-s3');
-import { Aws } from '@aws-cdk/core';
-import cdk = require('@aws-cdk/core');
+import { Bucket } from '@aws-cdk/aws-s3';
+import {
+	App,
+	Aws,
+	CfnParameter,
+	Construct,
+	Duration,
+	Stack,
+} from '@aws-cdk/core';
+import { NEWSLETTERS_BUCKET_NAME } from '../constants';
 
-export class LambdaService extends cdk.Stack {
-	constructor(scope: cdk.Construct, id: string) {
+export class LambdaCronService extends Stack {
+	constructor(scope: Construct, id: string) {
 		super(scope, id, { env: { region: Aws.REGION } });
 
-		const stage = new cdk.CfnParameter(this, 'Stage', {
+		const stage = new CfnParameter(this, 'Stage', {
 			type: 'String',
 			default: 'CODE',
 		});
 
-		const ssmPolicy = new iam.PolicyStatement({
+		const ssmPolicy = new PolicyStatement({
 			actions: ['ssm:GetParametersByPath'],
 			resources: [
-				`arn:aws:ssm:${Aws.REGION}:${Aws.ACCOUNT_ID}:parameter/frontend/${stage.valueAsString}/newsletters-api/*`,
+				`arn:aws:ssm:${Aws.REGION}:${Aws.ACCOUNT_ID}:parameter/frontend/${stage.valueAsString}/newsletters-source/*`,
 			],
 		});
 
-		const bucket = 'aws-frontend-artifacts';
-		const key = `frontend/${stage.valueAsString}/newsletters-api/newsletters-api.zip`;
-
-		const handler = new lambda.Function(this, 'frontend-newsletters-api', {
-			runtime: lambda.Runtime.NODEJS_14_X,
-			code: lambda.Code.fromBucket(
-				s3.Bucket.fromBucketName(this, 'lambda-code-bucket', bucket),
-				key,
-			),
-			handler: 'server.handler',
-			functionName: `frontend-newsletters-api-${stage.valueAsString}`,
-			memorySize: 384,
-			timeout: cdk.Duration.seconds(30),
-		});
+		const newslettersBucket = Bucket.fromBucketName(
+			this,
+			'newsletters-bucket',
+			NEWSLETTERS_BUCKET_NAME,
+		);
+
+		const lambdaCodeBucket = 'aws-frontend-artifacts';
+		const key = `frontend/${stage.valueAsString}/newsletters-source/newsletters-source.zip`;
+
+		const handler = new lambda.Function(
+			this,
+			'frontend-newsletters-source',
+			{
+				functionName: `frontend-newsletters-source-${stage.valueAsString}`,
+				runtime: lambda.Runtime.NODEJS_14_X,
+				memorySize: 384,
```

Comment:
could we explain this value `384`? I'm not sure where it comes from or why it has to be 384 instead of, for example, 383

Reply:
app takes around 310m of memory,  384 gives a bit of legging room, a bit magic. 
The default is 128  and is too low, minimum needs to be 310

Reply:
OK! That makes sense. What do you think about adding that as a comment here?